src/main/java/pl/onsight/wypozyczalnia/Application.java

```diff
@@ -1,18 +1,17 @@
 package pl.onsight.wypozyczalnia;
 
```

Comment:
ten import chba niepotrzebny :P

Reply:
to jakiś Tomcatowy tylko skąd on się tu trafił :D